portfolio/src/main/webapp/script.js

```diff
@@ -27,10 +27,38 @@ function addRandomQuote() {
       Flying blindly into the abyss, \
       believing therein lie the answers to the mysteries of the universe.'];
 
-  // Pick a random greeting.
+  // Pick a random quote.
   const quote = quotes[Math.floor(Math.random() * quotes.length)];
 
   // Add it to the page.
   const quoteContainer = document.getElementById('quote-container');
   quoteContainer.innerText = quote;
 }
+
+function getComment() {
+  fetch('/leave-comment').then(response => response.json()).then((comments) => {
```

Comment:
can you handle error case? e.g. 
`.catch((error) => {
  console.error('Error:', error);
});`
[full api](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch)